core/src/migrations/000066-addExportStatesAndRetry.ts

```diff
@@ -0,0 +1,72 @@
+import { Op } from "sequelize";
+
+export default {
+  up: async function (migration, DataTypes) {
+    await migration.sequelize.transaction(async () => {
+      await migration.addColumn("exports", "state", {
+        type: DataTypes.STRING(191),
+        allowNull: true,
+      });
+
+      await migration.bulkUpdate(
+        "exports",
+        { state: "failed" },
+        { state: null, errorMessage: { [Op.ne]: null } }
+      );
```

Comment:
TODO: If there are Exports mid-retry with a first error, we should put them into the `pending` state, not `failed`.  Any suggestions for a good query to determine this?

Reply:
I don't think this can be done... that's one of the previous problems of relying on resque for retries :/